e2e/tests/odoo-openmrs-flows.spec.ts

```diff
@@ -167,16 +150,12 @@ test('Discontinuing a synced OpenMRS drug order removes the corresponding Odoo q
 
 test('Ordering a drug with a free text medication dosage for an OpenMRS patient creates the corresponding Odoo customer with a filled quotation.', async ({ page }) => {
   // replay
-  await homePage.initiateLogin();
-  await expect(page).toHaveURL(/.*home/);
-  await homePage.createPatient();
-  await homePage.startPatientVisit();
-  await homePage.prescribeFreeTextMedicationDosage();
+  await openmrs.prescribeFreeTextMedicationDosage();
```

Comment:
Now knowing that there would be a `createDrugOrder()`, should this be `createDrugOrderWithFreeTextDosage()`?